src/append/mod.rs

```diff
@@ -261,8 +268,8 @@ mod test {
         ];
 
         for (input, expected) in test_cases {
-            let res = super::env_util::expand_env_vars(input.into());
-            assert_eq!(res, expected)
+            let res = super::env_util::expand_env_vars(input);
+            assert_eq!(res, expected.to_string_lossy())
```

Comment:
since we're now dealing with strings, all these tests should drop PathBuf::from and use static strs directly, then you can:
`assert_eq!(&res, expected)`

Reply:
Removed all the PathBuf::from stuff. Now everything is strings.